lib/piper/permissions/piper_rule_parser.yrl

```diff
@@ -42,6 +42,8 @@ command_selector ->
                                                        ?AST("BinaryExpr"):new('$4', [{left, Lhs},
                                                                                      {right, '$5'}]).
 permission_selector ->
+  permission_criterion : '$1'.
```

Comment:
Also I think this means `must have allow` is also allowed due to the rule below. Not a huge deal, but it might be a little weird to allow rules to be written that way.